loleaflet/src/layer/tile/CalcTileLayer.js

```diff
@@ -1086,6 +1094,38 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 		return this._selectedPart;
 	},
 
+	_createCommentStructure: function (menuStructure) {
+		var rootComment;
+		var annotations = this._annotations[this._selectedPart];
+
+		for (var i in annotations) {
+			rootComment = {
+				id: 'comment' + annotations[i]._annotation._data.id,
+				enable: true,
+				data: annotations[i]._annotation._data,
+				type: 'rootcomment',
+				text: annotations[i]._annotation._data.text,
+				annotation: annotations[i]._annotation,
+				children: []
+			};
+			rootComment.annotation.leafletId = annotations[i]._leaflet_id;	// required to highlight the selected cell
+			menuStructure['children'].push(rootComment);
+		}
+	},
+
+	_addHighlightSelectedWizardComment: function(annotation) {
+		if (this.lastWizardCommentHighlight) {
+			this.lastWizardCommentHighlight.css('background' , '');
+		}
+		this.lastWizardCommentHighlight = $(this._map._layers[annotation.leafletId]._container);
+		this.lastWizardCommentHighlight.css('background' , 'rgba(119, 119, 119, 0.25)');
```

Comment:
I think better to do this kind of stuff by adding some class. Then we can set color according to current theming in CSS.